src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -197,7 +197,13 @@ private void Cleanup(HostService hostService)
                          group file by p).ToList();
 
             var toHandleItems = files.Where(s => s.Key != null);
-            var notToHandleItems = files.Where(s => s.Key == null).SelectMany(s => s).Where(s => s.Type != DocumentType.Overwrite).ToList();
+            var notToHandleItems = files
+                .Where(s => s.Key == null)
+                .SelectMany(s => s)
+                .Where(s => s.Type != DocumentType.Overwrite &&
+                    !s.File.EndsWith(".yaml.md", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
is this a breaking change?

Reply:
Yes. If a markdown ends with ".y(a)ml.md", and the company ".y(a)ml" exists, it will not be processed. It's a fairly rare case.

Reply:
How about reporting some warning when markdown engine is not `markdig` and files with `.y(a)ml.md` are detected?

Reply:
Sounds good. Will add it in another PR, after qinezh's markdig extension is merged into docfx code base.